### DISCUSSION TITLE
[QEMU] Request: allow `--accel=hvf` by default (set `com.apple.security.hypervisor` entitlement)

### DISCUSSION THREAD

**Discussion by AkihiroSuda** [Homebrew/discussions#1442] [Everyday usage] [answered]
Running qemu with `--accel=hvf` on macOS 11 requires `com.apple.security.hypervisor` entitlement: https://www.arthurkoziel.com/qemu-on-macos-big-sur/

It would be great if the brew formula for QEMU can enable `com.apple.security.hypervisor` entitlement by default.
https://github.com/Homebrew/homebrew-core/blob/master/Formula/qemu.rb

**Comment by carlocab** [comment]
Please open a PR to enable this so it can be reviewed properly.

**Comment by AkihiroSuda** [comment, marked as answer]
This seems resolved in the latest revision

```console
$ codesign -d --entitlements :- /usr/local/Cellar/qemu/6.0.0/bin/qemu-system-x86_64 
Executable=/usr/local/Cellar/qemu/6.0.0/bin/qemu-system-x86_64
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
    <key>com.apple.security.hypervisor</key>
    <true/>
</dict>
</plist>
```
